### DISCUSSION TITLE
running 'git-filter-repo --version' and '<full path>/git-filter-repo --version' returns hexadecimal value on WSL

### DISCUSSION THREAD

**Discussion by ate26x** [newren/git-filter-repo#550] [General]
python version: 3.10.12
git-filter-repo version : 2.38.0 (according to the system message when package installed successfully)
OS : windows 11
editor: WSL (ubuntu) on VS Code

Tried uninstalling and reinstalling, it returns the same. Why would these commands return a hexadecimal value instead of 2.38.0 ?

**Comment by ate26x** [comment]
this works:
`pip3 show git-filter-repo | grep "Version:"`

these commands don't:
`git-filter-repo --version`

`cat /path/to/git-filter-repo/METADATA | grep "Version:"`

```
cd /path/to/git-filter-repo
cat __init__.py | grep "__version__"
```

**Reply by gdlxn-ibm** [reply]
I'm also seeing this on my Ubuntu 20.04 system.
